jenkins/test-history/gen_history

```diff
@@ -27,13 +27,12 @@ readonly jsonpath="gs://${bucket}/logs/$(date +%F).json"
 # generated.
 gsutil -q cp "${jsonpath}" "tests.json" || true
 
-# Create JSON report
-time python gen_json.py \
-  --jobs_dirs gs://kubernetes-jenkins/logs/ gs://kube_azure_log/ gs://rktnetes-jenkins/logs/ \
-  "--match=^kubernetes|kubernetes-build|kubelet-gce-e2e-ci"
-
-# Create static HTML reports out of the JSON
-python gen_html.py --output-dir=static --input=tests.json
+docker run --rm -v "$(pwd):/test-history" -w="/test-history" python:2.7 \
+    pip install -r requirements.txt && \
+    time python gen_json.py \
```

Comment:
this isn't executing inside the docker container. you want something like:

    docker run -i --rm -v "$(pwd):/test-history" -w="/test-history" python:2.7 bash <<EOF
    pip install -r requirements.txt && \
    time python gen_json.py ...
    EOF